src/toil/batchSystems/mesos/__init__.py

```diff
@@ -44,4 +44,8 @@
     # A dictionary with additional environment variables to be set on the worker process
     'environment',
     # A named tuple containing all the required info for cleaning up the worker node
-    'workerCleanupInfo'))
+    'workerCleanupInfo',
+    # A string containing the resolved mesos master address
```

Comment:
I used the word "resolved" because the value comes from running resolve(masterAddress)